code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -202,6 +202,9 @@ obj/structure/bed/Destroy()
 		if(C.handcuffed)
 			to_chat(user, SPAN_DANGER("You cannot buckle someone who is handcuffed onto this bed."))
 			return
+		if(isXeno(C) && C.stat == DEAD && (world.time < (C.timeofdeath + 2 MINUTES)))
```

Comment:
Mind that this was part of the original intelwebs capture experiment, not of techwebs or defcon, so you might not want to keep it for what you're doing